cgroups/fs/apply_raw.go

```diff
@@ -44,6 +44,7 @@ type subsystem interface {
 }
 
 type Manager struct {
+	sync.Mutex
```

Comment:
nit: I prefer to declare the mutex as `mu sync.Mutex` so Lock and Unlock doesnt belong to the Manager public API.